airflow/providers/microsoft/azure/hooks/data_lake.py

```diff
@@ -509,3 +509,18 @@ def delete_directory(self, file_system_name: FileSystemProperties | str, directo
         """
         directory_client = self.get_directory_client(file_system_name, directory_name)
         directory_client.delete_directory()
+
+    def test_connection(self):
+        """Test ADLS Gen2 Storage connection."""
+        success = (True, "Successfully connected to ADLS Gen2 Storage.")
```

Comment:
Why this extra variable instead of simply returning it directly…?

Reply:
True. Not required. Will make the change.

Reply:
Done.